### ISSUE TITLE
Hide connections not used in any steps of a transformation or job

### ISSUE THREAD

**Issue by joaociocca** [rpbouman/kettle-cookbook#71]
I just noted that, because I share connections on PDI, I end up with all my connections showing up in all my transformations and jobs - and also in their cookbooked versions. So filtering out connections not actually used would be nice.

**Comment by joaociocca** [comment]
hm... Variables could be filtered out like this too. Considering there's already a way to check if they're used in any step...

**Comment by rpbouman** [comment]
Good point. I suppose we could even mark them as unused and allow the user to decide whether they should be displayed;
